homeassistant/helpers/llm.py

```diff
@@ -20,8 +20,8 @@
 LLM_API_ASSIST = "assist"
 
 PROMPT_NO_API_CONFIGURED = (
-    "If the user wants to control a device, tell them to edit the AI configuration and "
-    "allow access to Home Assistant."
+    "Only if the user wants to control a device, tell them to edit the AI configuration "
```

Comment:
I know from your chat message why you did this but it would be good to document why in the PR description for others. Or preferably put it in a separate PR.

Reply:
https://github.com/home-assistant/core/pull/117935